packages/api-bindings/src/graphql/utils/isValidHandle.ts

```diff
@@ -0,0 +1,5 @@
+const validationRegex = /^[a-z](?:[a-z0-9_]{4,30})$/;
```

Comment:
Previously we have been using this one
```
/(@[a-z\d-_.]{1,31})/g;
```
was there any change in the underlying protocol handle validation rules?

Reply:
The `@` does not apply in this circumstance, also the `.` is part of the deployment suffix in Provide v1 (i.e. `.test` on test-net and `.lens` on main-net).

I am not sure if it can starts with number (maybe it can, I vaguely remember seeing few test handles) or `_`.

Reply:
I remember you can have as many `.` as you can, even `pawel....pawel`

Reply:
I am receiving mixed messages about this. Will double check and submit a follow up PR to tweak the regexp.